GuildWarsCalculator/src/main/java/com/theironyard/Controllers/GuildWarsApiController.java

```diff
@@ -0,0 +1,73 @@
+package com.theironyard.Controllers;
+
+import com.theironyard.Entities.GuildWarsRecipe;
+import com.theironyard.Entities.Item;
+import com.theironyard.Services.GuildwarsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by Nigel on 7/25/16.
+ */
+@Controller
+public class GuildWarsApiController {
+
+    @Autowired
+    RestTemplate restTemplate;
+
+    @Autowired
+    GuildwarsService guildwarsService;
```

Comment:
Why do you have this if you are going to use the `restTemplate`?  The service should be used to abstract out the connection and restTemplate making the controller easier.